developer-docs-site/docs/reference/move.md

```diff
@@ -0,0 +1,155 @@
+
+The Aptos Move adapter features include:
+* Fine grained storage that decouples the amount of data stored in an account affecting the gas fees for transactions associated with the account
+* [Tables](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/Table.move) for storing key, value data within an account at scale
+* Parallelism via [Block-STM](https://medium.com/aptoslabs/block-stm-how-we-execute-over-160k-transactions-per-second-on-the-aptos-blockchain-3b003657e4ba) that enables concurrent execution of transactions without any input from the user
+
+The Aptos framework ships with many useful libraries:
+* A [Token standard](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/Token.move) that makes it possible to create NFTs and other rich tokens without publishing a smart contract
+* A [Coin standard](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/Coin.move) that makes it possible to create type-safe Coins by publishing a trivial module
+* An [iterable Table](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/IterableTable.move) that allows for traversing all the entries within a table
+* A staking and delegation framework
+* A [`type_of`](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/TypeInfo.move) service to identify at run-time the address, module, and struct name of a given type
+* Multi-signer framework that allows multiple `signer` entities
+* A [timestamp service](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/Timestamp.move) that provides a monotonically increasing clock that maps to the actual current unixtime
+
+With much more coming soon...
+
+## Key Concepts in Move
+
+* Data should be stored within the account that owns it not the account that published the module.
+* Data flow should have minimal constraints with an emphasis on ecosystem usability
+* Prefer static type-safety over run-time safety via generics
+* A `signer` should be required to restrict access to adding or removing assets to an account unless it is explicitly clear
+
+### Data ownership
+
+Data should be stored within the account that owns it not the account that published the module.
+
+In Solidity, data is stored within the namespace of the account that created the contract. Typically this is represented by a map of an address to a value or of an instance id to the address of the owner.
+
+In Solana, data is stored within a distinct account associated with a contract.
+
+In Move, data can be stored within the module owner's account, but that creates the issue of ownership ambiguity and implies that two issues:
+1. It makes ownership ambiguous as the asset has no resource associated with the owner
```

Comment:
the ownership is based on the account address having the resource.  No quite follow why there is ambiguity. maybe some concrete examples can better explain the idea?

Reply:
The point is that some folks program in a solidity programming style and push data into the contract account